public/locales/en-US/translations.json

```diff
@@ -394,6 +394,18 @@
   "is_burned": "Burned",
   "fee_rate": "Fee Rate",
   "last_affecting_transaction": "Last affecting tx",
+  "Version": "Version",
   "increased_by": "increased by",
-  "Version": "Version"
+  "amm_lp_token_balance": "LP Token Balance",
+  "amm_token_balance": "Token Balance",
+  "trading_fee": "Trading Fee",
+  "amm_tvl": "TVL",
+  "amm_account_address": "Account Address",
```

Comment:
These keys don't need to be preceded with `amm_` - it makes it more general to not have that

Reply:
nit & not a blocker - lets skip

Reply:
I'd rather handle these now, because we'll forget about it later and they're pretty quick to resolve

Reply:
They are quick to resolve but theyve also existed since the first revision of the PR. This review has gone on for a long time and each iteration brings up more nits that have existed since the first revision. 

Theyre also not blocking - I think the best thing to do here is to get this change out for the user.  @ckniffen

Reply:
I dont think these need to be update at this time.